.gitignore

```diff
@@ -12,4 +12,5 @@ testconfig.json
 /package.json
 
 // Private Tarballs
-*.tgz
\ No newline at end of file
+*.tgz
+*.snap
```

Comment:
Snapshots are part of our unittesting.  If git shows that they are modified, then that means either the deviceData.json or the translator output changed.  If that was expected, then submit the new version.  It's a test to guarantee we don't accidentally add or remove anything from the translator output.